src/core/client/admin/components/ModerateCard/ModerateCard.tsx

```diff
@@ -231,14 +233,16 @@ const ModerateCard: FunctionComponent<Props> = ({
             >
               {commentBody}
             </CommentContent>
-            <div className={styles.viewContext}>
-              <Button iconLeft variant="text" onClick={viewConversationClick}>
-                <ButtonIcon>question_answer</ButtonIcon>
-                <Localized id="moderate-comment-viewConversation">
-                  <span>View conversation</span>
-                </Localized>
-              </Button>
-            </div>
+            {onConversationClick && (
+              <div className={styles.viewContext}>
+                <Button iconLeft variant="text" onClick={viewConversationClick}>
```

Comment:
No need to wrap this in a `useCallback` because it's already passed in the right form. Just use `onClick={onConversationClick}` instead!